resource-managers/kubernetes/core/src/test/scala/org/apache/spark/scheduler/cluster/k8s/ExecutorLifecycleTestUtils.scala

```diff
@@ -30,7 +30,7 @@ object ExecutorLifecycleTestUtils {
       .editOrNewStatus()
         .withPhase("failed")
         .addNewContainerStatus()
-          .withName("spark-executor")
+          .withName("spark-kubernetes-executor")
```

Comment:
Is this name change required based on the filtering that was added above or just for clarity when debugging?

Reply:
This is needed because of the filtering added above.